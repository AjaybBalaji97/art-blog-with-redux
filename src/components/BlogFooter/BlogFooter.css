footer {
    background: #000;
    color: #fff;
    position: fixed;
    bottom: 0;
    width:100%;
  }

  .footer p {
    margin-left:5%;
    font-size: 0.8em;  
  }