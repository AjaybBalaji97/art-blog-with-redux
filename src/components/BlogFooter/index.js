import React, { Component } from 'react';
import './BlogFooter.css';

class BlogFooter extends Component {
  render() {
    return (
        <footer className="footer">
        <p>&copy;artifact.com 2019</p>
      </footer>
    );
  }
}

export default BlogFooter;