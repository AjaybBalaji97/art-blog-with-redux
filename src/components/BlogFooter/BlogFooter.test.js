import React from 'react';
import renderer from 'react-test-renderer';
import BlogFooter from '.';
import {shallow} from 'enzyme';


describe('BlogHeader',()=>{
    it('should render header',()=>{
        const tree=renderer.create(<BlogFooter  />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});