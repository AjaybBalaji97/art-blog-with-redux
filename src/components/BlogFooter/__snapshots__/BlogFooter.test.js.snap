// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogHeader should render header 1`] = `
<footer
  className="footer"
>
  <p>
    Â©artifact.com 2019
  </p>
</footer>
`;
